### PR TITLE
[UPD] Localisation for Croatia

### PR DESCRIPTION
**Pull request by badbole**
Description of the issue/feature this PR addresses:

- as of 1.1.2023 Croatia official currency will be EUR (HRK left for historic purposes and usage until end of year)
- chart of accounts updated to latest version
- added account.group template - according to latest chart of accounts groups
- account report.lines updated to latest tax report structure
- taxex added to match tax report positions and new chart of accounts
- fiscal positions added according to basic accounting needs

Current behavior before PR:
- obsolete and hardly usable tax schema, chart of accounts
- 
Desired behavior after PR is merged:
New tax schena and tax report positions valid and usable

NOTE:

This PR need review and modification of data , currently it contains all old data as well as new settings to be tested

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by badbole** [comment]
@tsb-odoo could you please check and validate this module data and suggest acceptable data structure

File: addons/l10n_hr/demo/demo_company.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <odoo>
     <record id="partner_demo_company_hr" model="res.partner">
         <field name="name">HR Company</field>
-        <field name="vat">HR18724543544</field>
+        <field name="vat">HR01234567896</field>
```
Comment (badbole) [inline addons/l10n_hr/demo/demo_company.xml:5]: Using generic TAX ID - non existing but pass validation

File: addons/l10n_hr/data/l10n_hr_chart_data.xml [hunk 2]
```diff
@@ -10,8 +12,8 @@
             <field name="bank_account_code_prefix">101</field>
             <field name="cash_account_code_prefix">102</field>
             <field name="transfer_account_code_prefix">1009</field>
-            <field name="currency_id" ref="base.HRK"/>
-            <field name="code_digits">0</field>
+            <field name="currency_id" ref="base.EUR"/>
```
Comment (badbole) [inline addons/l10n_hr/data/l10n_hr_chart_data.xml:15]: Will be vaild starting 1.1.2023.
HRK should stay for at least one or two odoo releases before removing it from list of currencies

File: addons/l10n_hr/data/2021/res_country_group.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data ><!--noupdate="0"-->
+        <record id="country_group_rrif_europe-hr" model="res.country.group">
+            <field name="name">Europe (excluding HR)</field>
```
Comment (badbole) [inline addons/l10n_hr/data/2021/res_country_group.xml:5]: Might not be needed, as fiscal positions auto apply behives quite correct.

**Comment by badbole** [comment]
intended for odoo-16.0 branch - closing PR to master

**Comment by albertopoljak** [comment]
Hi, I can't find this merged nor in some new PR.
What's the status/plan for this?

**Comment by badbole** [comment]
> Hi, I can't find this merged nor in some new PR. What's the status/plan for this?

This PR is closed as it was made to master branch while odoo 16 was in master...
As meanwhile odoo16 branch was made, i will soon make a proper PR to branch 16.0
hopig to be merged soon... 

If you need sources they are still available on my fork

**Comment by badbole** [comment]
@albertopoljak new PR is created to designated branch 16.0:
https://github.com/odoo/odoo/pull/102018

**Comment by albertopoljak** [comment]
Ah I get it now, was a bit confused why it was closed.
Okay I'll move discussion there then.

**Comment by badbole** [comment]
> Ah I get it now, was a bit confused why it was closed. Okay I'll move discussion there then.

also.. check your gmail ;)

**Pull request closed without merging**